pivot/static/pivot/js/main.js

```diff
@@ -19,7 +19,10 @@ var statusLookupListener = [];
 
 /**** SETUP ****/
 if (window.location.pathname != "/about/" && window.location.pathname != "/login/") {
-    if (!~window.location.search.indexOf("?slow")) {
+    //indexOf will return a -1 if it doesn't find the string. ~ will take the bitwise not of the
+    //result, which will only be falsy if it is -1.
+    if (~window.location.search.indexOf("?slow") || ~window.location.search.indexOf("&slow")) {
```

Comment:
Believe this to be completely extraneous leftover code that can be removed in the future (after testing.) See GPS-423.